ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -295,8 +295,10 @@
         - pkts_num_fill_min='{{qp.wm_pg_shared_lossless.pkts_num_fill_min}}'
         - pkts_num_fill_shared='{{qp.wm_pg_shared_lossless.pkts_num_trig_pfc}}'
         - cell_size='{{qp.wm_pg_shared_lossless.cell_size}}'
+        - packet_size='{{qp.wm_pg_shared_lossless.packet_size}}'
```

Comment:
This will currently fail since vars/qos.yml doesn't have this parameter defined.

Reply:
Please remove the qos_sai.yml changes from this PR. you can add them in the new format once #1682 is merged

Reply:
Done.